Source/JavaScriptCore/inspector/JSInjectedScriptHost.cpp

```diff
@@ -315,7 +316,7 @@ JSValue JSInjectedScriptHost::getOwnPrivatePropertyDescriptors(JSGlobalObject* g
         if (!propertyName.string().startsWith('#'))
             continue;
 
-        result->putDirect(vm, Identifier::fromString(vm, String(propertyName.impl()->isolatedCopy())), objectConstructorGetOwnPropertyDescriptor(globalObject, object, propertyName));
+        result->putDirectIndex(globalObject, index++, Symbol::create(vm, *static_cast<SymbolImpl*>(propertyName.impl())));
```

Comment:
@WebKit/jsc-reviewers will this cause issues?  AFAICT `Symbol::finishCreation` will add the `SymbolImpl` to a global (per-`VM`) `symbolImplToSymbolMap`, with no place that it's ever removed.  should I add something special to prevent the addition, or is this OK?

Reply:
I think it should be fine because symbolImplToSymbolMap is a `WeakGCMap` so it should self clear when the `Symbol` gets GC'd.